react-charts/donut-chart/step4.md

```diff
@@ -1,31 +1,57 @@
-Now we'll add tooltips to the donut chart...
+Add a legend to the donut chart.
 
-## Task
+1) <strong>Make sure the App.js file is still open.</strong>
 
-1) Make sure the App.js file is still open
+2) <strong>Locate the code for the '<Chart>' component.</strong>

```

Comment:
Backticks instead of single quotes should be used (if the intent is to use the code highlighting in Katacoda)

Reply:
done